susi_python/main.py

```diff
@@ -134,26 +136,26 @@ def _get_rss_entities(self, data):
 
     def add_device(self, access_token, room_name):
 
-        get_device_info = self.api_endpoint + '/aaa/listUserSettings.json?'
-        add_device_url = self.api_endpoint + '/aaa/addNewDevice.json?'
+        get_device_info = self.base_url + '/aaa/listUserSettings.json?'
+        add_device_url = self.base_url + '/aaa/addNewDevice.json?'
         mac = get_mac()
         macid = ':'.join(("%012X"%mac)[i:i+2] for i in range(0,12,2))
 
         param1 = {
             'access_token':access_token
         }
 
-        # print(access_token)
+        logger.debug(access_token)
```

Comment:
Not sure how access token are used here, but isn't this creating the danger that a users shared the debug log and thus publicizes the access token?

Reply:
@norbusan , it was being used when we trying to ad offline/online support. I don't think we need it anymore. So , I should probably remove it.

Reply:
Yes please

Reply:
I think this change is still open @stealthanthrax